crates/forge/src/pretty_printing.rs

```diff
@@ -41,17 +41,23 @@ pub(crate) fn print_test_seed(seed: u64) {
     println!("{}: {seed}", style("Fuzzer seed").bold());
 }
 
-pub(crate) fn print_test_result(test_result: &TestCaseSummary, fuzzer_runs: Option<u32>) {
+pub(crate) fn print_test_result(test_result: &TestCaseSummary) {
+    if let TestCaseSummary::None {} = test_result {
+        return;
+    }
+
     let result_header = match test_result {
         TestCaseSummary::Passed { .. } => format!("[{}]", style("PASS").green()),
         TestCaseSummary::Failed { .. } => format!("[{}]", style("FAIL").red()),
         TestCaseSummary::Skipped { .. } => format!("[{}]", style("SKIP").yellow()),
+        TestCaseSummary::None {} => unreachable!(),
```

Comment:
Why do we have a test case like this at all if we are expecting to never handle it anyway?

Reply:
```
pub(crate) fn print_test_result(test_result: &TestCaseSummary) {
    if let TestCaseSummary::None {} = test_result {
        return;
    }

    let result_header = match test_result {
        TestCaseSummary::Passed { .. } => format!("[{}]", style("PASS").green()),
        TestCaseSummary::Failed { .. } => format!("[{}]", style("FAIL").red()),
        TestCaseSummary::Skipped { .. } => format!("[{}]", style("SKIP").yellow()),
        TestCaseSummary::None {} => unreachable!(),
    };
```

because I handled it in this `if` but rust still demand to have `TestCaseSummary::None` in the match statement

Reply:
What I meant is what's the purpose of the `TestCaseSummary::None`. I'm not sure why it was introduced.

Reply:
I will recheck purpose of it, because it was introduced before final version

Reply:
It handled the situation when one of a test returned Err. There are only tests finieshed before Err
![Zrzut ekranu 2023-10-9 o 12 33 22](https://github.com/foundry-rs/starknet-foundry/assets/15279784/4b8ee8c6-c645-4034-a5ec-c97dd991658a)

Reply:
I suggested renaming `None` to `InterruptedByError` or sth like that